cmd/telemeter-rhel-server/main.go

```diff
@@ -0,0 +1,395 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	stdlog "log"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/oklog/run"
+	"github.com/openshift/telemeter/pkg/tracing"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/spf13/cobra"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+	"go.opentelemetry.io/otel"
+	"golang.org/x/oauth2"
+
+	telemeter_http "github.com/openshift/telemeter/pkg/http"
+	"github.com/openshift/telemeter/pkg/logger"
+	"github.com/openshift/telemeter/pkg/receive"
+	"github.com/openshift/telemeter/pkg/server"
+)
+
+const desc = `
+Server for receiving Prometheus metrics through the remote_write API. Clients are authenticated 
+with mTLS.
+`
+
+func defaultOpts() *Options {
+	return &Options{
+		LimitBytes:        500 * 1024,
+		LimitReceiveBytes: receive.DefaultRequestLimit,
+		clusterIDKey:      "_id",
```

Comment:
in the context of rhel, this cluster id no longer makes sense so we can remove it

Reply:
done